src/style.css

```diff
@@ -17,6 +17,81 @@ li {
   list-style: none;
 }
 
+.content {
+  position: relative;
```

Comment:
- [ **OPTIONAL** ] Consider extracting all repeated CSS rules into a class and use them in the CSS. This ensures you implement clean and [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) code. Try this for `display: flex;`, `position: relative;`, `justify-content: center;`, `align-items: center;`, `flex-direction: column;`

**INSTEAD OF**
`position: relative;`

**USE**
`.p-relative {
    position: relative;
}`

**INSTEAD OF**
`display: flex;`

**USE**
`.d-flex {
    display: flex;
}`